packages/thirdweb/src/react/native/ui/icons/svgs.ts

```diff
@@ -37,6 +37,13 @@ export const DISCORD_ICON = `<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink
 </g>
 </svg>`;
 
+export const FARCASTER_ICON = `<svg width={width} height={height} viewBox="0 0 1000 1000" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
wait so we have SVG and base64 of (assuringly) the same svg? - can we simplify this somehow? (not new to this but I just realized it)